src/point_one/fusion_engine/messages/configuration.h

```diff
@@ -1282,6 +1288,16 @@ enum class MessageRate : uint8_t {
    * platforms.
    */
   INTERVAL_10_S = 12,
+  /**
+   * Output this message at this interval. Not supported for all messages or
```

Comment:
This is consistent with the existing ones, but might be a good idea to edit all of these to say more like "Output this message once every 30 seconds." and similar. It's fairly apparent from the enum name, but better to be explicit.